content/docs/design/DynamicConcurrency.md

```diff
@@ -0,0 +1,150 @@
++++
+title = "Dynamic Concurrency Requirement Overview and Design"
+description = "This page provides an overview of `Dynamic Concurrency Requirement`"
+weight = 10
+toc = true
+aliases = ["/docs/design/"]
+[menu.main]
+  parent = "Design"
+  weight = 2 
++++
+
+# **Dynamic Concurrency feature in KubeGene**
+
+In the process of gene data processing, it is often necessary to read the contents of a file to control concurrent tasks, or to obtain the "output results" of another tasks. 
```

Comment:
How about change `gene data processing` to a professional phrase like `genome sequencing`

Reply:
changed